modules/apps/object/object-test/src/testFunctional/tests/macros/ObjectPortlet.macro

```diff
@@ -12,6 +12,20 @@ definition {
 			locator1 = "ObjectPortlet#ENTRY_VALUE");
 	}
 
+	macro assertFilterResultsSpecificOrder {
+		var resultOrder = "0";
+
+		for (var entry	: list "${entry}") {
+			var resultOrder = ${resultOrder} + 1;
+
+			AssertVisible(
+				key_entry = "${entry}",
+				locator1 = "ObjectPortlet#ENTRY_VALUE",
+				resultOrder = "${resultOrder}",
+				value1 = "${entry}");
+		}
```

Comment:
Hi @PauloVAS ,

It is just asserting for the entry no matter which order it is.

You need to create a new path, with some name like: ENTRY_VALUE_SPECIFIC_ORDER

where it will have the key_entry var and also the the key_resultOrder.

See example of `ACInterests#INTEREST_VIEW_SPECIFIC_ORDER` that is used in `ACInterests.viewIndividualInterestSpecificOrder`